eq-author-api/schema/resolvers/base.js

```diff
@@ -1549,6 +1550,7 @@ const Resolvers = {
           id === sectionId && !pageId && !folderId
       ),
     comments: ({ id }, args, ctx) => ctx.comments[id],
+    allowRepeatingSection: (section) => Boolean (findIndex(getPagesFromSection(section), {pageType: "ListCollectorPage"})<0),
```

Comment:
```suggestion
    allowRepeatingSection: (section) =>
      findIndex(getPagesFromSection(section), {
        pageType: "ListCollectorPage",
      }) < 0,
```
Prettier hasn't been run on this line. This statement will return a boolean value, so `Boolean` can be removed